scvi/nn/_base_components.py

```diff
@@ -399,7 +412,7 @@ def forward(
         px_scale = self.px_scale_decoder(px)
         px_dropout = self.px_dropout_decoder(px)
         # Clamp to high value: exp(12) ~ 160000 to avoid nans (computational stability)
-        px_rate = torch.exp(library) * px_scale  # torch.clamp( , max=12)
+        px_rate = size_factor * px_scale  # torch.clamp( , max=12)
```

Comment:
I know it seems weird, but this is a breaking change now because this class assumed library was on the log scale...